notary-api/src/operations/notarize.js

```diff
@@ -0,0 +1,59 @@
+import uuidv4 from 'uuid/v4';
+import { addNotarizationJob } from '../queues/notarizationsQueue';
+import { notarizationResults } from '../utils/stores';
+import { sha3 } from '../utils/wibson-lib/cryptography/hashing';
+import { packPayData } from '../blockchain/batPay';
+
+const createNotarization = ({
+  orderId,
+  sellers,
+  callbackUrl,
+  notarizationPercentage = 0,
+  notarizationFee = 0,
+}) => {
+  const masterKey = uuidv4();
+  const payData = packPayData(sellers.map(s => s.sellerId));
+  const payDataHash = sha3(payData);
+  const lock = sha3(payDataHash.concat(masterKey));
+  const notarization = {
+    request: {
+      orderId,
+      sellers,
+      callbackUrl,
+    },
+    result: {
+      sellers,
```

Comment:
`orderId` is not necessary and `sellers` should contain the results.
i.e.
```suggestion
      sellers: sellers.map(({ sellerId }) => ({ sellerId, result: 'ignored' })),
```
(For now)

Reply:
I'm doing all the validation logic within the queue's job, the operation just starts the process.